.changeset/slimy-berries-mate.md

```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/vercel": patch
+---
+
+Fixes an issue with `sharp` by upgrading `@vercel/nft`
```

Comment:
What's the issue?

Reply:
This error at runtime `Could not load the "sharp" module using the linux-x64 runtime`. I don't know much we should put in the changeset

Reply:
What about "Fixes an issue that was preventing the use of `sharp` in some cases and causing a runtime error."